tools/schemacode/bidsschematools/tests/test_validator.py

```diff
@@ -343,6 +368,15 @@ def test_bids_datasets(bids_examples, tmp_path):
     # Have all files been validated?
     assert len(result["path_tracking"]) == 0
 
+    # Is the schema version recorded correctly?
+    schema_path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        "../data/schema",
+    )
+    with open(os.path.join(schema_path, "BIDS_VERSION")) as f:
+        expected_version = f.readline().rstrip()
+    assert result["bids_version"] == expected_version
```

Comment:
We seem to do this a lot just to validate. I think we could do the following in `__init__.py`:

```Diff
 version_file = files("bidsschematools.data") / "schema" / "SCHEMA_VERSION"
 with as_file(version_file) as path:
     __version__ = path.read_text().strip()
+
+bids_version_file = files("bidsschematools.data") / "schema" / "BIDS_VERSION"
+with as_file(bids_version_file) as path:
+    bids_version = path.read_text().strip()
```

It's done once on startup, allows access to the package and any downstream packages, and this block becomes:

```suggestion
    import bidsschematools as bst
    assert result["bids_version"] == bst.bids_version
```